### PR TITLE
Simple Health Check working 

### PR DESCRIPTION
**Pull request by SahasR**
If additional functionality required please label issue as WIP

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,6 +44,8 @@
     "husky": "^5.1.3",
     "mongoose": "^5.11.7",
     "morgan": "^1.10.0",
+    "nodejs-health-checker": "^1.1.6",
+    ""
```
Comment (Shehanka) [inline package.json:48]: @SahasR what is this ""

**Review by Shehanka — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (SahasR) [reply package.json:48]: A Node Module used that can be called to a HealthCheck

Comment (SahasR) [reply package.json:48]: Wait a minute what is it doing inside package.json -_-

**Comment by SahasR** [comment]
Athi yanthan checks pass wuna

**Review by Shehanka — APPROVED** [review]

**Pull request merged**